### PR TITLE
Fabric 1.19.2 Large Connector and Small Light Connector

### PR DESCRIPTION
**Pull request by skippyall**
This pull request implements the Large Connector and the Small Light Connector for 1.19.2. It is a draft because there is an issue with the behavior of energy networks.

### CODE REVIEW

**Comment by mrh0** [comment]
is this done?

**Comment by skippyall** [comment]
It is working, but when you break a connector, the network usage goggle tooltip is buggy and the Redstone Relay and the old Accumulator don't work. I will try to fix this before marking this as complete.

**Comment by skippyall** [comment]
I finally found the bug that broke it all. This should now be finished. I'll maybe make a separate PR for the Redstone Relay and the Accumulator.

**Comment by skippyall** [comment]
Seems like the problem with the Accumulator and the Redstone Relay was sitting in front of the screen. I didn't get that the "Push" side is the side that you should push your energy into, rather than the side that the Redstone Relay pushes its energy (Although I must say that this goggle tooltip is not so helpful)

File: src/main/java/com/mrh0/createaddition/energy/WireType.java [hunk 1]
```diff
@@ -67,4 +68,18 @@ public int getBlue() {
...
+	public static WireType of(Item item) {
+		if(item == CAItems.COPPER_SPOOL.get())
+			return WireType.COPPER;
+		if(item == CAItems.GOLD_SPOOL.get())
+			return WireType.GOLD;
+		if(item == CAItems.FESTIVE_SPOOL.get())
+			return WireType.FESTIVE;
+		//if(item == CAItems.IRON_SPOOL.get())
+		//	return WireType.IRON;
+		//if(item == CAItems.ELECTRUM_SPOOL.get())
```
Comment (mrh0) [inline src/main/java/com/mrh0/createaddition/energy/WireType.java]: Why didnt you add electrum? Did you not pull from Forge?

Comment (skippyall) [reply src/main/java/com/mrh0/createaddition/energy/WireType.java]: No I didn't. Does that even work with so many different changes?

Comment (skippyall) [reply src/main/java/com/mrh0/createaddition/energy/WireType.java]: I tried and there are about 1000 conflicts. I don't know if pulling from Forge is realistic...

Comment (skippyall) [reply src/main/java/com/mrh0/createaddition/energy/WireType.java]: If you wanted to do that, you'd better use architectury, I think.

Comment (skippyall) [reply src/main/java/com/mrh0/createaddition/energy/WireType.java]: Never mind, I can do that. ~~(However, it would still save a lot of work)~~

**Comment by skippyall** [comment]
How many commits could it have been?

**Comment by skippyall** [comment]
I think I am done.

**Comment by skippyall** [comment]
@mrh0  (In case you don't see it)

**Review by mrh0 — APPROVED** [review]

**Comment by skippyall** [comment]
Looking at the code of the Liquid Blaze Burner, I think it is quite buggy. I already fixed the capacity (Was just 49mb) and the Mechanical Arm Interaction in the last commit, but there still is a problem with manually clicking on it with a Bucket when it's full. There might also be a problem with portable fluid tanks.

**Comment by skippyall** [comment]
The problem with fluid tanks might exist on Forge too.

**Comment by skippyall** [comment]
Blazes now blaze correctly, they even eat Blaze Cakes with a straw in their mouth.

**Comment by mrh0** [comment]
Arte you done with this now?

**Comment by skippyall** [comment]
I haven't found any bugs, so I think this can be merged.

**Comment by mrh0** [comment]
Excellent!

File: src/main/java/com/mrh0/createaddition/blocks/liquid_blaze_burner/LiquidBlazeBurnerBlockEntity.java [hunk 2]
```diff
@@ -206,7 +211,7 @@ public void tick() {
 		if (remainingBurnTime > 0)
 			remainingBurnTime--;
 
-		if (activeFuel == FuelType.NORMAL || activeFuel == FuelType.SPECIAL)
+		if (activeFuel == FuelType.NORMAL)
```
Comment (TurnrDev) [inline src/main/java/com/mrh0/createaddition/blocks/liquid_blaze_burner/LiquidBlazeBurnerBlockEntity.java:214]: I think this line change might be responsible for #726

**Pull request merged**